packages/render-fidelity-tools/test/config.json

```diff
@@ -317,7 +326,8 @@
       },
       "renderSkybox": true,
       "exclude": [
-        "dspbr-pt"
+        "dspbr-pt",
+        "stellar"
```

Comment:
I must have misplaced my comment above. It was intended here: 
>This is one of the most important baseline cases; is there some reason stellar can't handle it?

Reply:
MetalRoughSpheresHDR added